packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -28,6 +28,13 @@ export const styles = css`
         align-items: center;
     }
 
+    :host(.nested-parent-row-first-cell) .cell-view {
+        padding-left: calc(
+            ${mediumPadding} + ${controlHeight} + ${controlHeight} *
+                (var(--ni-private-table-cell-nesting-level) - 1)
```

Comment:
The subtraction of 1 here is essentially where the logic moved from the `updateCellIndentLevels` method in the `Row` class. It's possible that the `updateCellIndentLevels` method can now be removed entirely, but that is something I'd rather explore outside of this PR.